### DISCUSSION TITLE
Add a clear button in app

### DISCUSSION THREAD

**Discussion by jojojojojoj5564656465465** [PrivateBin/PrivateBin#1378] [Ideas]
I would like a clear screen the input field or a home button instead.

Thank you

**Comment by elrido** [comment]
Could you describe how that home button would differ from the "New" button? Or do you mean three different buttons with different functions instead of just "New" which does all three actions?

**Reply by jojojojojoj5564656465465** [reply]
On Android I only have "ENCRYPT AND SHARE" Could you make on top left i litle home button to reset everything

**Reply by elrido** [reply]
That doesn't sound like your instance is using any of the templates we ship with privatebin (which is fine, thats what templates are for). Could you share the URL of your current instace, just to verify that it indeed is a privatebin instance? You can compare yours to the [demo instance](https://privatebin.net) or one from the [instance directory](https://privatebin.info/directory/).
